### ISSUE TITLE
fix 'should you update a variable' quetsions on replay

### ISSUE THREAD

**Issue by noamoss** [hasadna/reportit-agent#65]
tried on https://admin.equality.org.il/#/report/185.

1. I started to answer a report (initial intake). entered some info (like the involved company name).
2. clicked back to the main dashboard. The example report appeared there with  'בטיפול' status + 2 open tasks, 
3.  I clicked on it, the conversation ran from the beginning (not as a replay).
4.  I went back to the main dashboard, and clicked again on the same report.. The same happened - I had to answer the questions to continue, so  I clicked on the first two buttons to move on. Then I clicked on the menu button to go back to the main dashboard, again
5. clicked on the same report again, and this time, the display stopped where I got in phase 4 (i.e. waiting for my 3rd answer). As you can see on the screenshot my original company name was saved, but I am being asked again, instead of just showing be the last call 

![image](https://user-images.githubusercontent.com/3581741/70833388-4bdf9a80-1e00-11ea-8ae9-f42197121da4.png)

it might be worth to mention this check was made right after using the script editor for the first time, implementing the required change on #50.

**Comment by akariv** [comment]
@noamoss did you try this after fixing the bug in the script?

**Comment by noamoss** [comment]
@akariv just did, so now it works as expected in terms of not asking for the input again, but it runs gradually, and not like it was supposed to according to #64 - the thread runs again, in a slow phase, instead of showing it all at once

(https://admin.equality.org.il/#/report/187):
![image](https://user-images.githubusercontent.com/3581741/70852268-2dcd7500-1ea8-11ea-9407-ae640ceca599.png)

**Comment by noamoss** [comment]
(you might want to close this issue, in favor of #64, fine by me)

**Comment by noamoss** [comment]
@akariv reopenning this issue, as a similar problem appeared: 

1. after answering 'no' to a question:
![image](https://user-images.githubusercontent.com/3581741/70870239-6ac56480-1f99-11ea-944e-8f449547a7fe.png)

2. I click back to the dashboard, and enter the report again, the replay works until the same question:
![image](https://user-images.githubusercontent.com/3581741/70870254-9ba59980-1f99-11ea-96a2-2bc97579bad3.png)

so now we are not talking about the fact it does not follow #64. 
Not sure if helps, but my guess is the problem occurs because we are still using a function to ask and save the response of the agent, which prevent you from replaying it.

**Comment by akariv** [comment]
Yes, replay doesn't happen for questions initiated from inside functions.
Once you finish getting rid of them, please reopen if there's still a
problem.

**Comment by noamoss** [comment]
@akariv blocked by https://github.com/hasadna/reportit-agent/issues/70
please reassign to me when ready for check

**Comment by noamoss** [comment]
the main problem here is solved, but a minor side effect was found, to be reported in a separate issue: https://github.com/hasadna/reportit-agent/issues/74
